controllers/user.js

```diff
@@ -61,38 +79,47 @@ async function add(req, res) {
   });
 }
 
-const bundleVerifiedEmails = async(email, emailSha1, ifPrimary, allBreaches) => {
+const bundleVerifiedEmails = async(email, emailSha1, ifPrimary, id, verificationStatus, allBreaches) => {
   const foundBreaches = await HIBP.getBreachesForEmail(emailSha1, allBreaches, true);
 
   const emailEntry = {
     "email": email,
     "breaches": foundBreaches,
     "primary": ifPrimary,
+    "id": id,
+    "verified": verificationStatus,
   };
+
   return emailEntry;
 };
 
-async function getDashboard(req, res) {
-  const allBreaches = req.app.locals.breaches;
-  const user = req.session.user;
+const getAllEmailsAndBreaches = async(user, allBreaches) => {
```

Comment:
Maybe not in this PR, but we should consolidate "user" and "subscriber" into `subscriber`. Could be follow-up.